components/dashboard/src/data/organizations/org-members-query.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { TeamMemberInfo } from "@gitpod/gitpod-protocol";
+import { useQuery } from "@tanstack/react-query";
+import { useMemo } from "react";
+import { publicApiTeamMembersToProtocol, teamsService } from "../../service/public-api";
+import { useCurrentTeam } from "../../teams/teams-context";
+import { useCurrentUser } from "../../user-context";
+
+type UseOrganizationMembers = {
+    organizationId: string;
+};
+type OrganizationMembersQueryResult = TeamMemberInfo[];
+
+export const useOrganizationMembers = ({ organizationId }: UseOrganizationMembers) => {
```

Comment:
I debated having a query for the team (and not scoped to team members), but there's some thinking through how we incorporate react-query with how we load teams, or single team records and what the TeamContext currently does, so I didn't want to conflate that for now. We can always adjust this later if we end up with a team query to use instead.